doc/api/process.md

```diff
@@ -1819,6 +1819,46 @@ process.report.writeReport();
 
 Additional documentation is available in the [report documentation][].
 
+## process.resourcesUsage()
```

Comment:
Can you name this `resourceUsage()` (resource instead of resources) throughout the PR.

Reply:
Wouldn't it make more sense to keep `resources` since there are metrics related to different resources (io, cpu and memory)?

Reply:
@vmarchaud I think it’s just that noun+noun compounds in English generally use the singular form for the all but the last word/head of the phrase. So, if you really wanted to emphasize the fact that this function provides multiple sets of information, you could use `resourceUsages()`, but I’d prefer Colin’s suggestion and just go with the singular. And if nothing else, consistency with `memoryUsage()` is valuable, too.

Reply:
Oh i see, english is not my main english so i didn't knew about that.